help/en/html/doc/Technical/GitFAQ.shtml

```diff
@@ -104,7 +102,7 @@ on how to
 	Once you have a copy of your changes on GitHub, it is easy to 
 	<a href="https://help.github.com/articles/creating-a-pull-request/">generate a pull request</a> (link to GitHub):
 	<ul>
-	<li> In a browser, navigate to the repository on GitHub that has the changes you want someone else to pull and
+	<li> In a browser, navigate to your repository on GitHub that has the changes you want someone else to pull and
```

Comment:
You can generate a PR from a repository you don't own, so "the" is more correct here (for example, I could generate a PR from your repository if I felt the need to do so.